source/_integrations/http.markdown

```diff
@@ -215,6 +215,13 @@ $ curl -X GET -H "Authorization: Bearer LONG_LIVED_ACCESS_TOKEN" \
 }
 ```
 
+To delete the sensor, send DELETE request with curl
+
+```bash
+$ curl -X DELETE -H "Authorization: Bearer LONG_LIVED_ACCESS_TOKEN" \
+       http://localhost:8123/api/states/binary_sensor.radio
```

Comment:
I get that you are adding this, but that said... one should really not be using this, as it messes with the state machine. I would advise building an integration that actually provides entities and not using these REST APIs to modify values in the state machine directly.

Reply:
Hi @frenck! Can you elaborate on this please? 

How is this different from MQTT which is used by many integrations and has same capabilities of registering sensors, updating their state and deleting them…

In my case it's a super-custom LLM-based home microservice from which I expose few sensors and doing it over HTTP REST API is so much easier than MQTT which I wasted hours with not to mention building a whole custom integration…

The reason why I'm adding `DELETE` docs is because silly me hit the REST API with some silly_test_sensor_name/test_state just to see if it works and it worked but I didn't want to restart HA to get rid of that synthetic test sensor 😅

I mean of course if HA removes this REST API I'll have to do MQTT or Integration, but I'm sure there are thousands of users like me who prefer the simplicity of REST API for their homebrew things :)